examples/experimental/horizontal-pod-autoscaling/README.md

```diff
@@ -0,0 +1,207 @@
+If load balancers are not supported (e.g.: on Vagrant), we can expose php-apache service as ``ClusterIP`` and connect to it using the proxy on the master:
+
+```console
+$ kubectl expose rc php-apache --port=80 --type=ClusterIP
+NAME         LABELS           SELECTOR         IP(S)     PORT(S)
+php-apache   run=php-apache   run=php-apache             80/TCP
+
+$ kubectl cluster-info | grep master
+Kubernetes master is running at https://146.148.6.215
+
+$ curl -k -u admin:password https://146.148.6.215/api/v1/proxy/namespaces/default/services/php-apache/
+OK!
+```
+
+
+## Step Two: Create horizontal pod autoscaler
+
+Now that the server is running, we will create a horizontal pod autoscaler for it.
+To create it, we will use [hpa-php-apache.yaml](hpa-php-apache.yaml) file, which look like this:
+
+```yaml
+apiVersion: experimental/v1alpha1
+kind: HorizontalPodAutoscaler
+metadata:
+  name: php-apache
+  namespace: default
+spec:
+  maxReplicas: 10
+  minReplicas: 1
+  scaleRef:
+    kind: ReplicationController
+    name: php-apache
+    namespace: default
+  target:
+    quantity: 100m
+    resource: cpu
+```
+
+Please be aware that it sets the target CPU consumption to 100 milli-cores, while in [rc-php-apache.yaml](rc-php-apache.yaml) each pod requests 200 milli-cores.
+As a general rule, the autoscaler's target should be lower than the request.
+Otherwise, overloaded pods may not be able to consume more resources than requested what will prevent scaling-up.
+
+We will create the autoscaler by executing the following command:
+
+```console
+$ kubectl create -f ./examples/experimental/horizontal-pod-autoscaling/hpa-php-apache.yaml
+horizontalpodautoscaler "php-apache" created
+```
+
+We may check the current status of autoscaler by running:
+
+```console
+$ kubectl get experimental/hpa
+NAME         REFERENCE                                   TARGET     CURRENT   MINPODS   MAXPODS   AGE
+php-apache   ReplicationController/default/php-apache/   100m cpu   0 cpu     1         10        4m
+```
+
+Please not that the current CPU consumption is 0 as we are not sending any requests to the server.
+
+## Step Three: Increase load
+
+Now, we will see how the autoscaler reacts on the increased load of the server.
+We will start an infinite loop of queries to our server (please run it in a different terminal):
+
+```console
+$ while true; do curl http://146.148.6.244; done
+```
+
+We may examine, how CPU load was increased (the results should be visible after about 2 minutes) by executing:
+
+```console
+$ kubectl get experimental/hpa
+NAME         REFERENCE                                   TARGET     CURRENT    MINPODS   MAXPODS   AGE
+php-apache   ReplicationController/default/php-apache/   100m cpu   471m cpu   1         10        10m
+```
+
+In the case presented here, it bumped CPU consumption to 471 milli-cores.
+In affect, the replication controller was resized to 5 replicas:
```

Comment:
s/In affect/As a result/

BTW, in the next version of the code it would probably be good to add REPLICAS column to "get experimental/hpa" -- it is already in the HorizontalPodAutoscalerStatus so this should be trivial.